reposerver/repository/repository.go

```diff
@@ -3113,21 +3113,30 @@ func (s *Service) UpdateRevisionForPaths(_ context.Context, request *apiclient.U
 }
 
 func (s *Service) updateCachedRevision(logCtx *log.Entry, oldRev string, newRev string, request *apiclient.UpdateRevisionForPathsRequest, gitClientOpts git.ClientOpts) error {
-	repoRefs := make(map[string]string)
+	oldRepoRefs := make(map[string]string)
+	newRepoRefs := make(map[string]string)
+
+	oldCacheRev := oldRev
+	newCacheRev := newRev
```

Comment:
maybe I'm missing something but why do we need these assignments here?
`oldCacheRev` and `newCacheRev` are not used elsewhere and then they are assigned new values in lines `3135-3136`

Reply:
Ah yes , I didn't notice the other assignment is in the if block - thanks